src/test/components/ThreadActivityGraph.test.js

```diff
@@ -208,5 +208,63 @@ describe('ThreadActivityGraph', function() {
         'set globalCompositeOperation'
```

Comment:
nit: there are several typos in the commit log:

`the the` in the title
` draCalls` in the text